### DISCUSSION TITLE
launch mongosh in image mongo:6.0.4 and 6.0.5 throws an error node::Abort() [mongosh]

### DISCUSSION THREAD

**Discussion by AnjeiKozhev** [docker-library/mongo#625] [General]
when I run the command **mongosh** in the image, I get this error. 

docker run --entrypoint /bin/bash -it mongo:6.0.5

root@673609d0a744:/# mongosh
mongosh[8]: ../src/node_platform.cc:61:std::unique_ptr<long unsigned int> node::WorkerThreadsTaskRunner::DelayedTaskScheduler::Start(): Assertion `(0) == (uv_thread_create(t.get(), start_thread, this))' failed.
 1: 0xb81280 node::Abort() [mongosh]
 2: 0xb812fe  [mongosh]
 3: 0xbed0ee  [mongosh]
 4: 0xbed1b6 node::NodePlatform::NodePlatform(int, v8::TracingController*) [mongosh]
 5: 0xabe90f node::MultiIsolatePlatform::Create(int, v8::TracingController*) [mongosh]
 6: 0xab5c84 main [mongosh]
 7: 0x7fa14a426d90  [/lib/x86_64-linux-gnu/libc.so.6]
 8: 0x7fa14a426e40 __libc_start_main [/lib/x86_64-linux-gnu/libc.so.6]
 9: 0xab860e  [mongosh]
Aborted (core dumped)

This also applies to image 6.0.4. In images 6.0.3 and below, this is not observed. Accordingly, I cannot start the container with my configuration file for mongodb. What could be the problem? Thank you.

**Comment by tianon** [comment]
I can't actually reproduce: :grimacing:

```console
$ docker run -it --rm --pull=always mongo:6.0.5 bash
6.0.5: Pulling from library/mongo
Digest: sha256:928347070dc089a596f869a22a4204c0feace3eb03470a6a2de6814f11fb7309
Status: Image is up to date for mongo:6.0.5
root@b849d7c422fa:/# mongosh
Current Mongosh Log ID:	646276d55af328bf39914bec
Connecting to:		mongodb://127.0.0.1:27017/?directConnection=true&serverSelectionTimeoutMS=2000&appName=mongosh+1.8.2
MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017
```

Unfortunately, we do not have the bandwidth to provide in-depth integration/deployment/environment debugging or support here; these sorts of questions/requests would be more appropriately posted to a dedicated support forum, such as [the Docker Community Slack](https://dockr.ly/comm-slack), [Server Fault](https://serverfault.com/help/on-topic), [Unix & Linux](https://unix.stackexchange.com/help/on-topic), or [Stack Overflow](https://stackoverflow.com/help/on-topic).

**Reply by AnjeiKozhev** [reply]
Thanks for the reply, I figured out the problem. Maybe someone will come in handy. The thing is that I had Docker-ce version 20.10.8 installed, after upgrading to version 23.0.5 the images mongo 6.0.4 and 6.0.5 started working correctly. It may be worth adding this information somewhere in the system requirements, in any case, I did not find such information.
